### PR TITLE
Messages

### PR DESCRIPTION
**Pull request by IdanTo1**

### CODE REVIEW

File: src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    /**
+     * sets the serial number of the Moneypenny who acquired the agents.
+     * @param _MoneypennySerial Moneypenny's serial
+     */
+    public void setMoneypennySerial(int _MoneypennySerial) {
+        this._MoneypennySerial = _MoneypennySerial;
+    }
+
```
Comment (Hroy02) [inline src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java]: Parameters should be without underscore (`_`) right?

File: src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
+package bgu.spl.mics.application.messages;
+
+import java.util.List;
+
+public class AgentsAvailableObject {
+    private List<String> _agentsSerials;
+    private List<String> _agentsNames; //TODO maybe only agents, event with serial and future with names?
+    private int _missionDuration;
```
Comment (Hroy02) [inline src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java]: I think this is fine. either way you should probably drop the comment

**Review by Hroy02 — CHANGES_REQUESTED** [review]
Small fixes needed

File: src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java (hunk 2 above)
Comment (IdanTo1) [reply src/main/java/bgu/spl/mics/application/messages/AgentsAvailableObject.java]: it was for you to read (: i'll remove it

**Review by Hroy02 — APPROVED** [review]
Perfect! You can merge it

**Pull request merged**